packages/adblocker/src/filters/cosmetic.ts

```diff
@@ -748,16 +748,22 @@ export default class CosmeticFilter implements IFilter {
       return undefined;
     }
 
-    const args = parts.slice(1).map((part) => {
-      if (
-        (part.startsWith(`'`) && part.endsWith(`'`)) ||
-        (part.startsWith(`"`) && part.endsWith(`"`))
-      ) {
-        return part.substring(1, part.length - 1);
-      }
-      return part;
-    });
-
+    const args = parts
+      .slice(1)
+      .map((part) => {
+        if (
+          (part.startsWith(`'`) && part.endsWith(`'`)) ||
+          (part.startsWith(`"`) && part.endsWith(`"`))
+        ) {
+          return part.substring(1, part.length - 1);
+        }
+        return part;
+      })
+      .map((part: string) => {
+        return part
+          .replace(new RegExp(/\\u002C/, 'g'), ',')
```

Comment:
Can you make the regexp a constant? To avoid that it gets recomputed all the time.